### DISCUSSION TITLE
Suggestions don't work -- what am I doing wrong!?

### DISCUSSION THREAD

**Discussion by kellygold** [zed-industries/zed#7538] [Feature Requests]
Hi -- hoping someone can clear up requirements for Copilot suggestions to work

I'm using the default settings (no changes to my `.config/zed/settings.json` file), where it seems `how_copilot_suggestions: true` is the default. 

When I type in my editor, I tried in Python, and Javascript and on 2 machines -- I get no auto-complete suggestions

I also see this in the command palette, but it does nothing when I try to access it. Should this do something? 

![image](https://github.com/zed-industries/zed/assets/28990947/cb0c2e36-a97f-4d6c-b7d5-7a143a62e8aa)

I would expect to see "something" be suggested by copilot but nada. Only suggestions I get is the helpers from other packages. 

![image](https://github.com/zed-industries/zed/assets/28990947/d4932084-0aa8-4c71-80b2-277b9ee09bab)

I am guessing this is user error, but I could really use some guidance!

**Comment by kellygold** [comment]
few suspicious entries in the log 
```
2024-02-07T18:43:44-07:00 [ERROR] Cancelled LSP request task for "signInConfirm" id 5 which took over 120s
2024-02-07T18:43:44-07:00 [ERROR] crates/copilot/src/copilot.rs:115: LSP request timeout
2024-02-07T18:43:44-07:00 [ERROR] crates/copilot/src/copilot.rs:115: LSP request timeout
2024-02-07T18:43:44-07:00 [INFO] 5 unhandled notification
```

**Comment by Moshyfawn** [comment]
Could you try either Preview [`v0.128.1`](https://zed.dev/releases/preview/0.128.1) or Stable [`0.127.4`](https://zed.dev/releases/stable/0.127.4) and see if you still get this problem? There's been some work done on the Copilot integration that might have fixed it.

**Comment by souraOP** [comment]
Only copilot suggestions are coming up but no autocompletion for Java, C++ or Python.. none of them of are coming.. I think integration of the JDK inside Zed is not yet done

**Comment by JamieMason** [comment]
Incase it helps anyone else, I had this on Zed 0.183.10 and running `copilot: reinstall` from the command palette fixed it for me.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
